pages/docs/scale/docker_swarm.html

```diff
@@ -0,0 +1,171 @@
+title: Docker Swarm
+author: Chris Ward
+
+This documentation was tested with Docker Machine version `0.7.0` and Docker `1.11.0`. Please let us know if you experience any problems.
+
+Swarm is Docker's native clustering tool, turning a pool of Docker instances into one single abstract host. This makes it perfect for a scalable database like Crate. Generally Swarm uses instances created by Docker machine, but can also be [setup manually](https://docs.docker.com/swarm/install-manual/).
+
+To get a Crate cluster running on Swarm on a local machine you can follow [the Swarm getting started guide](https://docs.docker.com/swarm/install-w-machine/) and replace `docker run hello-world` with Crate commands:
+
+```bash
+docker run -P -d crate
+docker run -P -d crate
+docker run -d -p 4200:4200 -p 4300:4300 crate
+```
+
+Resulting in something like:
+
+![Docker Swarm]({% media '/media/1604/swarm-crate.png' %})
+
+## Production Example
+
+This production-ready example will deploy a swarm cluster to Google Compute Engine.
+
+### Allow Traffic
+
+First open the Firewall ports that the Crate cluster will need:
+
+```bash
+gcloud compute firewall-rules create default-ssh --allow tcp:22,icmp --source-range 0.0.0.0/0
+gcloud compute firewall-rules create default-swarm --allow tcp:2376,tcp:3376 --source-range 0.0.0.0/0
+gcloud compute firewall-rules create default-crate --allow tcp:4200,tcp:4300 --source-range 0.0.0.0/0
+```
+
+### Create Master
+
+Create a bash script called _startup-master.sh_ and add the following to it:
```

Comment:
@chaudum Yup, that keeps throwing me, my editor defaults to '_' (As do most editors…)